from app.Exceptions.GeneralExceptions import GeneralExceptions


class ValidationException(GeneralExceptions):
    pass
